java/configuration/src/main/java/sleeper/configuration/properties/instance/EMRServerlessProperty.java

```diff
@@ -30,7 +31,7 @@ public interface EMRServerlessProperty {
             .propertyBuilder("sleeper.bulk.import.emr.serverless.architecture")
             .description("The architecture for EMR Serverless to use. X86_64 or ARM64 (Coming soon)")
             .propertyGroup(InstancePropertyGroup.BULK_IMPORT).defaultValue("X86_64")
```

Comment:
You can use `EmrInstanceArchitecture.X86_64.toString()` for the default value here, just to be a bit clearer that this string is the same as the validation predicate

Reply:
Updated as requested